lib/addons/helm-addon/index.ts

```diff
@@ -23,4 +25,20 @@ export abstract class HelmAddOn implements spi.ClusterAddOn {
         const chart = {...this.props, ...{ values, wait, timeout, createNamespace} };
         return kubectlProvider.addHelmChart(chart);
     }
-}
\ No newline at end of file
+}
+//below is my addon approach method for when I get too it. Leaving for later
+function validateInputAddonConstraints(props: HelmAddOnProps){
```

Comment:
Why is this function here? We should drop it  and call the generic function from the contraint utils

Reply:
That was from before we did the generic approach and I never deleted it, good point.